ocaml/alerts/expiry_alert.ml

```diff
@@ -0,0 +1,167 @@
+(*
+ * Copyright (C) Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+module XenAPI = Client.Client
+module Date = Xapi_stdext_date.Date
+
+type message_name_t = string
+
+type message_priority_t = int64
+
+type message_id_t = message_name_t * message_priority_t
```

Comment:
Will the `message_priority_t`be used? I can't find its usage in this module.

Reply:
No, I added this type definition just to make the type definition of `message_id_t` clear. I think this is clearer than comment.